src/data/constants.ts

```diff
@@ -64,4 +64,19 @@ export const wordsPerPage = 20;
 export const wordsPerUnit = 600;
 export const pagesNum = 30;
 
-export const MIN_PASSWORD_LENGTH = 8;
\ No newline at end of file
+export const MIN_PASSWORD_LENGTH = 8;
```

Comment:
I would like to suggest to use namespaces for collections of variables that belongs sematically to defined entity. 
I assume that current file is not yet fullfilled.

Reply:
> namespaces

Suggestion is added to the "To do" tasks list.